core-bundle/src/Resources/contao/library/Contao/Controller.php

```diff
@@ -77,7 +77,7 @@ public static function getTemplate($strTemplate)
 			/** @var PageModel $objPage */
 			global $objPage;
 
-			if ($objPage->templateGroup)
+			if (null !== $objPage && $objPage->templateGroup)
```

Comment:
The typehint three lines above should also be adjusted to `/** @var PageModel|null $objPage */`.

Reply:
That would need to be done everywhere though (there are 40 more occurrences). Imho that should be a separate PR.

Reply:
Done. It also makes sense there...